server/mail/templates/mfa.html

```diff
@@ -0,0 +1,181 @@
+            margin: 0;
+            padding-bottom: 32px;
+        }
+
+        a {
+            text-decoration: none;
+            color: #6a67fe;
+        }
+
+        a:hover {
+            text-decoration: none;
+        }
+
+        @media only screen and (max-device-width: 480px) {
+            table {
+                width: 100% !important;
+                padding: 0 !important;
+                margin: 0 !important;
+            }
+
+            td {
+                width: 100% !important;
+                padding: 20px !important;
+            }
+        }
+    </style>
+</head>
+<body style="color: #192147">
+<table
+        align="center"
+        border="0"
+        cellpadding="0"
+        cellspacing="0"
+        height="100%"
+        width="100%"
+        bgcolor="#F9FAFC"
+        style="
+        background: #f9fafc;
+        font-family: 'Nunito Sans', sans-serif;
+        border-collapse: collapse;
+      "
+>
+    <tr>
+        <td valign="top" align="center">
+            <table
+                    width="580"
+                    align="center"
+                    cellpadding="0"
+                    cellspacing="0"
+                    bgcolor="#ffffff"
+                    style="
+              margin: 20px 20px;
+              border: 1px solid #e2e4ea;
+              border-radius: 8px;
+            "
+            >
+                <tr>
+                    <td
+                            colspan="2"
+                            bgcolor="#ffffff"
+                            style="
+                  padding-top: 40px;
+                  padding-left: 48px;
+                  font-family: 'Nunito Sans', sans-serif;
+                  border-radius: 8px 8px 0px 0px;
+                "
+                    >
+                        <a href="https://fleetdm.com" target="_blank">
+                            <img
+                                    alt="Fleet logo"
+                                    src="{{.AssetURL}}/fleet-logo-blue-118x41@2x.png"
+                                    style="height: 41px; width: 118px"
+                            />
+                        </a>
+                    </td>
+                </tr>
+                <tr>
+                    <td
+                            colspan="2"
+                            style="
+                  padding-top: 48px;
+                  padding-bottom: 48px;
+                  padding-left: 48px;
+                  padding-right: 48px;
+                  font-family: 'Nunito Sans', sans-serif;
+                "
+                    >
+                        <h1>Log in to Fleet</h1>
+                        <p>Hello <b>{{.FullName}}</b>,</p>
+                        <p>
+                            Please click the personalized link below to log into your account.
+                        </p>
+                        
+                        <a
+                                href="{{.BaseURL}}/login/mfa/{{.Token}}"
```

Comment:
Am guessing this GET endpoint is on the other frontend PR?

Reply:
Yep, that's my expectation (/cc @RachelElysia), since that's a FE landing route.